solutions/advanced/leetcode-96.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: leetcode-96
+source: Leetcode
+title: Unique Binary Search Trees
+author: Rameez Parwez
+---
+
+## Explanation
+
+The number of structurally unique Binary Search Trees (BST) for $n$ nodes is the $n^{th}$ catalan number.
```

Comment:
soooo you're just not gonna say anything about...why?

Reply:
some one did the same with  bracket sequence I so ig this will do

Reply:
added the proof of it you can go through it